StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Components/NavigationBar/LinkSheetNavigationBar.swift

```diff
@@ -98,9 +98,8 @@ class LinkSheetNavigationBar: SheetNavigationBar {
 
     override func createBackButton() -> UIButton {
         let image = Image.icon_chevron_left_standalone.makeImage(template: true)
```

Comment:
Ooh you know, we use system icons now for the liquid glass nav bar to exactly match the system. Maybe a follow- up.https://github.com/stripe/stripe-ios/pull/5544/files#diff-dab538710af695b014168dcd1a906083476596d0717dc3458de14748011beb74